docs/_static/references.bib

```diff
@@ -150,4 +150,28 @@ @book{church_murray_2018
   year={2018},
 }
 
```

Comment:
In line 114, volume is the same line of title. I think this is my fault. Could you edit this? Just hit enter LOL